features/accessibility/provider.feature

```diff
@@ -143,6 +142,8 @@ Feature: Provider accessibility
     And the page is accessible
     Then I choose "No"
     Then I click 'Save and continue'
+    Then I should be on a page showing 'Check your answers'
```

Comment:
Could this say `Then I should be on the 'check_income_answers' page showing 'Check your answers'`?

Reply:
Yep, good spot will update